packages/driver/src/dom/transform.ts

```diff
@@ -35,7 +35,7 @@ interface TransformInfo {
 const extractTransformInfoFromElements = ($el: any, list: TransformInfo[] = []): TransformInfo[] => {
   list.push(extractTransformInfo($el))
 
-  const $parent = $el.parent()
+  const $parent = $el.parent() || $el.offsetParent()
```

Comment:
since you've used offsetParent this way a lot, and its going to be inefficient/misuse of the method, maybe you should add a helper which returns the `parentNode` or shadow root's `host`. that would be a better method of getting the actual parent imo